.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,62 @@
+name: Replicated PR Channel Create
+
+on:
+  pull_request:
```

Comment:
What is the use case for this work flow? 
If the replicated tool is for self-hosted only then wouldnt the release.yaml workflow be sufficient?

Reply:
The idea with this would be to be able to have a deployable chart for every PR that can be used for testing. I wouldn't have included it in the first cutover but Replicated had a good example of this so I decided to pull it in. We don't have to necessarily include it (i.e., we can remove the pull_request trigger until we want to use this functionality)